File: ChatGPT/ChatGPT/c++/countAlphabet.cpp
#include <iostream>
#include <string>
#include <unordered_map>
#include <cctype>

int main() {
    std::string input;
    std::unordered_map<char, int> counts;

    std::cout << "알파벳을 입력하세요: ";
    std::getline(std::cin, input);

    for (char c : input) {
        if (std::isalpha(c)) {
            c = std::tolower(c);
            counts[c]++;
        }
    }

    std::cout << "입력한 알파벳의 개수:" << std::endl;
    for (char c = 'a'; c <= 'z'; ++c) {
        std::cout << c << ": " << counts[c] << std::endl;
    }

    return 0;
}

File: ChatGPT/ChatGPT/go/countAlphabet.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	var input string
	fmt.Print("알파벳을 입력하세요: ")
	fmt.Scanln(&input)

	counts := make(map[rune]int)

	for _, char := range strings.ToLower(input) {
		if char >= 'a' && char <= 'z' {
			counts[char]++
		}
	}

	fmt.Println("입력한 알파벳의 개수:")
	for char := 'a'; char <= 'z'; char++ {
		count := counts[char]
		fmt.Printf("%c: %d\n", char, count)
	}
}
